ansible/roles/content-snapshot-indexer/tasks/main.yml

```diff
@@ -0,0 +1,129 @@
+---
+- name: Create content-snapshot directory
+  file:
+    path: '{{ content_snapshot_path }}'
+    state: directory
+    mode: 0777
+  become: yes
+  become_user: "{{ analytics_user }}"
+  tags:
+    - deploy
+
+- name: Copy distributed jar
+  copy: src={{ content_snapshot_distribuction_name }} dest={{ content_snapshot_path }}
+  tags:
+    - deploy
+  
+- name: Ensure removing jars directory
+  file:
+    path: '{{ content_snapshot_path }}jars/'
+    state: absent
+  become_user: "{{ analytics_user }}"  
```

Comment:
Same - Not required. This task is running already as analytics user. Same comment for below blocks also.